debian/autobake-deb.sh

```diff
@@ -112,34 +105,40 @@ LOGSTRING="MariaDB build"
 CODENAME="$(lsb_release -sc)"
 EPOCH="1:"
 
-dch -b -D ${CODENAME} -v "${EPOCH}${UPSTREAM}${PATCHLEVEL}~${CODENAME}" "Automatic build with ${LOGSTRING}."
+dch -b -D "${CODENAME}" -v "${EPOCH}${UPSTREAM}${PATCHLEVEL}~${CODENAME}" "Automatic build with ${LOGSTRING}."
 
 echo "Creating package version ${EPOCH}${UPSTREAM}${PATCHLEVEL}~${CODENAME} ... "
 
-# On Travis CI, use -b to build binary only packages as there is no need to
-# waste time on generating the source package.
+# On Travis CI and Gitlab-CI, use -b to build binary only packages as there is
+# no need to waste time on generating the source package.
 if [[ $TRAVIS ]]
 then
   BUILDPACKAGE_FLAGS="-b"
 fi
 
+# Use eatmydata is available to build faster with less I/O, skipping fsync()
```

Comment:
who's fsyncing?

Reply:
Any step in the entire build. Eatmydata is a generic performance optimizer for use cased where errors are not critical, such when running a development build (if my laptop looses power during the build it does not matter, no critical data).

Reply:
Updated inline comment to be more clear about this